src/GAP.jl

```diff
@@ -17,6 +17,12 @@ import Base: length, convert, finalize
 
 import Libdl
 
+using GAPTypes
```

Comment:
Perhaps this would be more appropriate?
```suggestion
import GAPTypes: GapObj
```

Reply:
Actually, this is exactly the same, but I will change it.